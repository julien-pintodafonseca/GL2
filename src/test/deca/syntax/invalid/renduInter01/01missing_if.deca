// Description:
//      Programme lexicalement incorrect
//      => une instruction conditionnelle if...else doit d√©buter par une branche if
//
// Resultats:
//      Erreur syntaxique
//      Ligne 11 : token 'else' inattendu

{
    int x = 4;
    else (x == 4) {
        println("hello else_if2 : ", x);
    }

}
