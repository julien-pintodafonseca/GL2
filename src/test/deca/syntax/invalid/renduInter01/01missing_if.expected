01missing_if.deca:11:4: mismatched input 'else'
